libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/RouterWrapper.kt

```diff
@@ -286,6 +258,7 @@ class RouterWrapper(
     }
 
     override fun cancelAll() {
+        // should it also cancel the internal coroutines?
```

Comment:
I think yes. I think we should do even more. It would be great if we would be able to cancel a coroutine for an individual response. 

Right now getting route is a 2 steps async process: 
1. Request route
2. Parse response

If a user calls `mapboxNavigation.cancelRouteRequest` when the SDK parsing result he will get callback called anyway. This behaviour seems like a bug for me 🙂

Reply:
Yea, that's exactly what I noticed as well. I don't want to scope creep this PR, so I'll probably just cut a ticket for the bug.